cmd/install/install.go

```diff
@@ -229,3 +241,25 @@ func downloadArgoCD(osPlatform string) error {
 	}
 	return nil
 }
+
+func installCAPI(ver string) error {
```

Comment:
The CAPI operator is intended to be a declarative solution (that will be idempotent too) https://github.com/kubernetes-sigs/cluster-api-operator/blob/main/docs/capi-operator-proposal.md#motivation Haven't looked at it in more detail, there are YAMLs for cloud providers too - they will likely need customisation with some inputs coming from users e.g. which cloud providers to enable, keys for AWS if CAPA is enabled etc.  Shall we do this from a separate CLI and only verify for CAPI and providers within Arlon?

Reply:
Read Bich's comment above. We're going all the way

Reply:
Maybe, this looks very much customisable. Perhaps this is the better option